examples/node/pdf2svg.js

```diff
@@ -70,73 +70,70 @@ ReadableSVGStream.prototype._read = function () {
 };
 
 // Streams the SVG element to the given file path.
-function writeSvgToFile(svgElement, filePath) {
+async function writeSvgToFile(svgElement, filePath) {
   let readableSvgStream = new ReadableSVGStream({
     svgElement,
   });
   const writableStream = fs.createWriteStream(filePath);
-  return new Promise(function (resolve, reject) {
-    readableSvgStream.once("error", reject);
-    writableStream.once("error", reject);
-    writableStream.once("finish", resolve);
-    readableSvgStream.pipe(writableStream);
-  }).catch(function (err) {
+  try {
+    await readableSvgStream.once("error");
+    await writableStream.once("error");
+    await writableStream.once("finish");
+    await readableSvgStream.pipe(writableStream);
+  } catch (err) {
     readableSvgStream = null; // Explicitly null because of v8 bug 6512.
     writableStream.end();
     throw err;
-  });
+  }
```

Comment:
The `once` method is Node.js functionality which doesn't return a Promise, hence this unfortunately cannot work; please revert all changes in the function!